web/docs/data-model/backends.md

```diff
@@ -286,6 +287,11 @@ async function createUser(
 }
 ```
 
+Wasp exports a type called `DbSeedFn` which you can use to easily type your seeding function.
```

Comment:
Maybe say it with code:
```ts
type DbSeedFn = (prisma: PrismaClient) => Promise<void>
```

Reply:
I decided against this since we don't do it for other functions we provide. Plus, hiding the signature inner complexity (although there isn't much of it) is the point of the type.